### ISSUE TITLE
Lambda.select() methods vurnelable to null

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dschulten/lambdaj#50]
```
When iterable passed to select is null it causes NullPointerException. In v2.0 
it returned empty list which is a way nicer as does not require all those 
if(iterable != null) checks outside the select call.
```

Original issue reported on code.google.com by `dinad...@gmail.com` on 12 Nov 2010 at 5:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 2.3.2
```

Original comment by `mario.fu...@gmail.com` on 20 Dec 2010 at 9:41
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Great! Thank you!
```

Original comment by `dinad...@gmail.com` on 20 Dec 2010 at 10:23
